### PR TITLE
Small V+ documentation changes

### PR DESCRIPTION
**Pull request by GimpArm**
Changed all the V+ examples to have enforceMod=true so anyone copying the examples will immediately have a working setup.
Added a link to the complete V+ configuration documentation.

### CODE REVIEW

**Comment by InB4DevOps** [comment]
There are a few changes that do not require EnforceMod to be enabled (eg Password & datarate) so I wouldn't want to have the documentation changed where it's not needed..

File: README.md [hunk 1]
```diff
@@ -312,7 +312,7 @@ Example:
 ```
 -e VALHEIM_PLUS=true \
 -e VPCFG_Server_enabled=true \
--e VPCFG_Server_enforceMod=false \
```
Comment (InB4DevOps) [inline README.md:315]: This is wrong. For changing the datarate enforceMod must not be true.

File: README.md [hunk 2]
```diff
@@ -321,7 +321,7 @@ turns into `/config/valheimplus/valheim_plus.cfg`
 ```
 [Server]
 enabled=true
-enforceMod=false
```
Comment (InB4DevOps) [inline README.md:324]: see above

File: README.md [hunk 3]
```diff
@@ -640,12 +640,12 @@ To do so enable ValheimPlus (`VALHEIM_PLUS=true`) and configure the following se
...
```
Comment (InB4DevOps) [inline README.md:643]: see above

File: README.md [hunk 4]
```diff
@@ -640,12 +640,12 @@ To do so enable ValheimPlus (`VALHEIM_PLUS=true`) and configure the following se
...
+enforceMod=true
 dataRate=600
 ```
 (Or whatever `dataRate` value you require. The value is in kb/s with a default of 60.)
 
-Alternatively start with `-e VPCFG_Server_enabled=true -e VPCFG_Server_enforceMod=false -e VPCFG_Server_dataRate=600`.
```
Comment (InB4DevOps) [inline README.md:648]: see above

File: README.md [hunk 5]
```diff
@@ -654,10 +654,10 @@ To do so enable ValheimPlus (`VALHEIM_PLUS=true`), set an empty password (`SERVE
 ```
 [Server]
 enabled=true
-enforceMod=false
+enforceMod=true
 disableServerPassword=true
 ```
-Alternatively start with `-e VPCFG_Server_enabled=true -e VPCFG_Server_enforceMod=false -e VPCFG_Server_disableServerPassword=true`.
```
Comment (InB4DevOps) [inline README.md:660]: wrong as well

**Comment by GimpArm** [comment]
If there are so many instances where you don't need/want enforceMod=true, then someone who understands this a lot better than I do needs to describe better what this actually does and when you want to enabled/disable it. Which is why I originally suggested in my ticket that it should be documented better.

File: README.md (hunk 1 above)
Comment (GimpArm) [reply README.md:315]: This line is in the Translation table section and has nothing to do with datarate.

File: README.md (hunk 2 above)
Comment (GimpArm) [reply README.md:324]: also see above

**Comment by GimpArm** [comment]
V+ also doesn't really document it. It appears exactly one time in their github readme and zero times on their config documentation page. That's who should be adding this. I had assumed this was an undocumented flag you added since you asked me to document it here. When no one actually writes what this flag does in their documentation, how can a user write anything about it? Asked me to write what would have helped me, that would have been an example that actually shows how to turn V+ on and not "just turn some parts on" with no one documenting which parts are turned on and when.

I realize now my issue should have been with V+, thank you for your help.

**Pull request closed without merging**